bindings/cs/rl.net/LiveModel.cs

```diff
@@ -953,6 +1045,39 @@ public MultiSlotResponseDetailed RequestMultiSlotDecisionDetailed(string eventId
             return result;
         }
 
+        public bool TryRequestMultiSlotDecisionDetailed(string eventId, string contextJson, ActionFlags flags, out MultiSlotResponseDetailed response, int[] baselineActions, ApiStatus apiStatus)
+        {
+            response = new MultiSlotResponseDetailed();
+
+            GC.KeepAlive(this);
+            return this.TryRequestMultiSlotDecisionDetailed(eventId, contextJson, flags, response, baselineActions, apiStatus);
+        }
+
+        public bool TryRequestMultiSlotDecisionDetailed(string eventId, string contextJson, ActionFlags flags, MultiSlotResponseDetailed response, int[] baselineActions, ApiStatus apiStatus)
+        {
+            int result = LiveModelRequestMultiSlotDecisionDetailedWithBaselineAndFlags(this.DangerousGetHandle(), eventId, contextJson, (uint)flags, response.DangerousGetHandle(), baselineActions, apiStatus.ToNativeHandleOrNullptrDangerous());
+            return result == NativeMethods.SuccessStatus;
```

Comment:
You should add `GC.KeepAlive(this)` (and maybe `GC.KeepAlive(response)` before the return, though that is probably not necessary, since it should be on the stack in this call), since otherwise the GC might decide to deallocate the object while the request is in flight.